.github/workflows/spell-check.yml

```diff
@@ -14,24 +14,24 @@ jobs:
   spell-check:
     runs-on: ubuntu-latest
     container:
-      image: rocker/tidyverse:4.0.3
+      image: rocker/tidyverse:4.1.2
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       - uses: actions/checkout@v3
 
       - name: Install packages
-        run: Rscript --vanilla -e "install.packages('spelling', repos = c(CRAN = 'https://cloud.r-project.org'))"
+        run: Rscript --vanilla -e "install.packages(c('spelling'), repos = c(CRAN = '$CRAN'))"
```

Comment:
This is more of a question than a comment.

How does the action know the definition of `$CRAN`? Where is that defined?

Reply:
`$CRAN` is defined in the `rocker` image.